setup.py

```diff
@@ -30,6 +30,10 @@
     description="This is nwbwidgets, widgets for viewing the contents of a "
     "NWB-file in Jupyter Notebooks using ipywidgets.",
     install_requires=requirements,
+    extras_require={
+        'human_electrodes' : ['skspatial', 'nilearn', 'trimesh'],
+        'mouse_electrodes' : ['ccfwidget', 'aiohttp']
```

Comment:
```suggestion
        'full' : ['ccfwidget', 'aiohttp', 'skspatial', 'nilearn', 'trimesh']
```

We always like to have one of the 'extra' requirements to be an optional full installation if the user might want to be able to view any widget for any neurodata type